assets/css/layout.scss

```diff
@@ -328,7 +328,6 @@ code {
 .content ol {
   line-height: 1.5em;
   /* 24px/16px */
-  padding-left: 1.5em;
 
   @media (min-width: 940px) {
```

Comment:
need to be max-width, not min-width, and use 0 as value with !important. It should solve this issue in mobile. 

<img width="405" height="165" alt="image" src="https://github.com/user-attachments/assets/bbf56328-ad98-4054-adda-29f0e96b7a12" />